From 74b5459695a61ec855a89c410a84b1e53457c92c Mon Sep 17 00:00:00 2001
From: Janne Grunau <janne-libav@jannau.net>
Date: Sun, 1 Jun 2014 15:50:42 +0200
Subject: [PATCH 123/125] aarch64: add darwin style PAGE/PAGEOFF relocations

Signed-off-by: Linus Yang <laokongzi@gmail.com>
---
 libavutil/aarch64/asm.S | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/libavutil/aarch64/asm.S b/libavutil/aarch64/asm.S
index 8ac5c18..fb635d0 100644
--- a/libavutil/aarch64/asm.S
+++ b/libavutil/aarch64/asm.S
@@ -57,7 +57,10 @@ ELF     .size   \name, . - \name
 .endm
 
 .macro  movrel rd, val
-#if CONFIG_PIC
+#if CONFIG_PIC && defined(__APPLE__)
+        adrp            \rd, \val@PAGE
+        add             \rd, \rd, \val@PAGEOFF
+#elif CONFIG_PIC
         adrp            \rd, :pg_hi21:\val
         add             \rd, \rd, :lo12:\val
 #else
-- 
1.9.3

