From 6a4e82041397d1a1fb89b89ff0324662a60c7fa0 Mon Sep 17 00:00:00 2001
From: Janne Grunau <janne-libav@jannau.net>
Date: Wed, 23 Jul 2014 10:06:15 +0200
Subject: [PATCH 128/131] aarch64: use MACH-O const data asm directive in const
 macro

Signed-off-by: Linus Yang <laokongzi@gmail.com>
---
 libavutil/aarch64/asm.S | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/libavutil/aarch64/asm.S b/libavutil/aarch64/asm.S
index 3690a33..9ff3ecc 100644
--- a/libavutil/aarch64/asm.S
+++ b/libavutil/aarch64/asm.S
@@ -51,7 +51,11 @@ ELF     .type   \name, %function
 ELF     .size   \name, . - \name
         .purgem endconst
     .endm
+#ifndef __MACH__
         .section        .rodata
+#else
+        .const_data
+#endif
         .align          \align
 \name:
 .endm
-- 
2.1.0

