From af412d16f9c25c68aee33c5740a5057761cedf76 Mon Sep 17 00:00:00 2001
From: Michael Niedermayer <michaelni@gmx.at>
Date: Wed, 6 Aug 2014 13:59:18 +0200
Subject: [PATCH 129/131] avutil/cpu: add aarch64 entries to 2nd table

Signed-off-by: Michael Niedermayer <michaelni@gmx.at>
Signed-off-by: Linus Yang <laokongzi@gmail.com>
---
 libavutil/cpu.c | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/libavutil/cpu.c b/libavutil/cpu.c
index 4bae5bd..0842a55 100644
--- a/libavutil/cpu.c
+++ b/libavutil/cpu.c
@@ -169,6 +169,10 @@ int av_parse_cpu_caps(unsigned *flags, const char *s)
         { "vfp",      NULL, 0, AV_OPT_TYPE_CONST, { .i64 = AV_CPU_FLAG_VFP      },    .unit = "flags" },
         { "vfpv3",    NULL, 0, AV_OPT_TYPE_CONST, { .i64 = AV_CPU_FLAG_VFPV3    },    .unit = "flags" },
         { "neon",     NULL, 0, AV_OPT_TYPE_CONST, { .i64 = AV_CPU_FLAG_NEON     },    .unit = "flags" },
+#elif ARCH_AARCH64
+        { "armv8",    NULL, 0, AV_OPT_TYPE_CONST, { .i64 = AV_CPU_FLAG_ARMV8    },    .unit = "flags" },
+        { "neon",     NULL, 0, AV_OPT_TYPE_CONST, { .i64 = AV_CPU_FLAG_NEON     },    .unit = "flags" },
+        { "vfp",      NULL, 0, AV_OPT_TYPE_CONST, { .i64 = AV_CPU_FLAG_VFP      },    .unit = "flags" },
 #endif
         { NULL },
     };
-- 
2.1.0

